projects/aws/eks-anywhere-test/Makefile

```diff
@@ -15,8 +14,13 @@ REPO_NO_CLONE=true
 BUILD_TARGETS=local-images
 RELEASE_TARGETS=images
 
+HAS_HELM_CHART=true
+
 include $(BASE_DIRECTORY)/Common.mk
 
+.PHONY: clean-repo
```

Comment:
Instead of this how about we change the preq in Common.mk to something like

```
.PHONY: clean
clean: $(if has_no_repo....,clean-repo,)
```

Reply:
Done